core/tests/protractor/libraryPagesTour.js

```diff
@@ -45,7 +45,16 @@ describe('Library pages tour', function() {
 
   var visitRecentlyPublishedPage = function() {
     browser.get('library/recently_published');
-  }
+  };
+
+  it('visits the search page', function() {
```

Comment:
(I also moved this earlier so that it fails faster, thus allowing more retries if needed.)

Reply:
Yep. That should be better.